src/ml.py

```diff
@@ -58,7 +81,11 @@ def encode(self, text: str, device: Optional[str] = None) -> np.ndarray:
         return self.encoder.encode(text, device=device, show_progress_bar=False)
 
     def encode_batch(
-        self, text_batch: List[str], batch_size: int = 32, device: Optional[str] = None
+        self,
+        text_batch: List[str],
+        batch_size: int = 32,
+        device: Optional[str] = None,
+        experimental_use_sliding_window: bool = False,
```

Comment:
It looks like this is never set to `True`, so I'm a bit confused for the use of the sliding window functionality in general -- I was trying to understand in which cases would we use this instead of relying on the text-blocks, as you implemented above.

Reply:
is this resolved by the conversation below?

Reply:
it's set to `True` in `utils.encode_parser_output`. Does that resolve this?